app/models/miq_expression/component/is.rb

```diff
@@ -0,0 +1,17 @@
+class MiqExpression::Component::Is < MiqExpression::Component::Leaf
+  def to_arel
+    if target.date?
+      if MiqExpression::RelativeDatetime.relative?(value)
+        start_val = MiqExpression::RelativeDatetime.normalize(value, "UTC", _mode = "beginning").to_date
+        end_val = MiqExpression::RelativeDatetime.normalize(value, "UTC", _mode = "end").to_date
+        target.between(start_val..end_val)
+      else
+        target.eq(MiqExpression::RelativeDatetime.normalize(value, "UTC", _mode = "beginning").to_date)
+      end
+    else
+      start_val = MiqExpression::RelativeDatetime.normalize(value, timezone, _mode = "beginning").utc
```

Comment:
When I was first working on this I found these strings a bit opaque out of context. I don't think that "mode" adds that much on reflection - I can remove these